src/components/Video/Video.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
this is less a component and more a composition I feel since very specific to what it is suppose to do